src/Chapter1.hs

```diff
@@ -519,8 +522,15 @@ branches because it is an expression and it must always return some value.
 👩‍🔬 Due to lazy evaluation in Haskell, only the expression from the branch
   satisfying the check will be returned and, therefore, evaluated.
 -}
+
 closestToZero :: Int -> Int -> Int
-closestToZero x y = error "closestToZero: not implemented!"
+closestToZero x y = closest
+    where
+        absClosest :: Int
+        absClosest = min (abs x) (abs y)
+        closest    = if absClosest == abs x then x else y
```

Comment:
Very interesting solution! 💡 
Though it could be a bit simplified. You see that all you need is to compare the absolute value of both `x` and `y`, and based on that return the result (as you do in the `then` and `else` clauses 👍🏼